packages/lesswrong/components/tagging/TagPageRouter.tsx

```diff
@@ -4,6 +4,27 @@ import { Components, registerComponent } from "../../lib/vulcan-lib";
 import { useOverrideLayoutOptions } from "../hooks/useLayoutOptions";
 import { useTagBySlug } from "./useTag";
 
+export const getTagStructuredData = (tag: TagPageFragment | TagPageWithRevisionFragment) => {
+  const hasSubTags = !!tag.subTags && tag.subTags.length > 0;
+
+  return {
+    "@context": "http://schema.org",
+    "@type": "WebPage",
+    "name": tag.name,
```

Comment:
There is no description here (to avoid bloating the html), which should be fine because it's already the main content of the page